k8s.gcr.io/images/k8s-staging-slack-infra/images.yaml

```diff
@@ -1,15 +1,20 @@
 - name: slack-event-log
```

Comment:
gah, we should really delete these old promoter manifests, will get this moved over to the current one in registry.k8s.io/